AdamSensors/V1/test.yaml

```diff
@@ -0,0 +1,20 @@
+adam_ip: 140.252.33.160
```

Comment:
I think the name of the folder must be `v1` instead of `V1`.... 

Also, can you try to make the name of the file a bit more descriptive?